packages/db-ui-elements-stencil/stencil.config.ts

```diff
@@ -130,11 +130,11 @@ export const config: Config = {
           src: `${getWWWPath()}/node_modules/@db-ui/core/dist/css${
             getTheme() ? '/' + getTheme() : ''
           }/db-ui-core.vars.css`,
-          dest: 'static/css/db-ui-core.vars.css'
+          dest: (getTheme() ?? getTheme() + '/') + 'css/db-ui-core.vars.css'
```

Comment:
We only reference 'css/db-ui-core.vars.css' in https://github.com/db-ui/elements/blob/ade59ef9441c6630258b4711dca9cb76bab12d16/.storybook/preview-head.html

Why should we need `getTheme()` here? Moreover this would not work for local development

Reply:
@nmerget do we need it on the previous line 131 than still?

Reply:
just for the record: we're currently focusing on most likely to build an individual Storybook instance by "theme", so the `dest` path would always be the same even though that the source is different.